301-quarkus-vertx-kafka/pom.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
```

Comment:
pls change repository URL below to HTTPS; as maven 3.8.1 doesn't recognize insecure connections.

Reply:
Out of scope of this PR. Please create a separated PR to address this.